src/modules/pages/components/ExpenditurePage/LockedSidebar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo } from 'react';
+
+import { LockedExpenditureSettings } from '~dashboard/ExpenditurePage/ExpenditureSettings';
+import { LockedPayments } from '~dashboard/ExpenditurePage/Payments';
+import { Colony } from '~data/index';
+
+import { ValuesType } from './ExpenditurePage';
+import { ExpenditureTypes } from './types';
+
+const displayName = 'pages.ExpenditurePage.LockedSidebar';
+
+interface Props {
+  colony: Colony;
+  formValues?: ValuesType;
+}
+
+const LockedSidebar = ({ colony, formValues }: Props) => {
+  const { expenditure, recipients, filteredDomainId } = formValues || {};
+
+  const secondFormSection = useMemo(() => {
```

Comment:
I would prefer to have it directly in the html with just a ternary operator. No need to separate it out with the use memo.